client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -149,7 +149,8 @@ export const DateUtils = {
     fromUnixTime(Math.max(...dates.map(d => getUnixTime(d as Date)))),
   isPast,
   isFuture,
-  isExpired: (expiryDate: Date): boolean => isPast(expiryDate),
+  isExpired: (expiryDate: Date | string | number): boolean =>
```

Comment:
Changed the input type of this to match the input type of `isPast` (string dates are fine)